website/project/model.py

```diff
@@ -3169,6 +3229,11 @@ def add_authorizer(self, user, approved=False, save=False):
         return False
 
     def remove_authorizer(self, user):
+        """Remove a user as an authorizer
```

Comment:
Normally we try to avoid implicit saves on any method that alters the database (and instead prefer a `save=False` kwarg so that the user has to consciously choose to save).

That said, I'm unclear on how much of this class belongs to this PR, so it's possible that this may be out of scope. Not looking to trigger a refactor on a mountain of existing code.